### PR TITLE
Data clean script for notes section in json.

### PR DESCRIPTION
**Pull request by JacOng17**
Command to use to run the script: `./dataCleaned.rb > out.json`
The script takes the json files & looks for the reference number & notes section. It then puts the data into a json file & ignores the ones with null for the notes section.

### CODE REVIEW

File: dataCleaned.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+require 'json'
+
+path = './txt_cleaned'
+files = Dir["#{path}/*.json"]
+
+results = []
+
+files.each do |filename|
+  # puts filename
```
Comment (Br3nda) [inline dataCleaned.rb]: you can probably remove this debug statement now

File: dataCleaned.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  text = File.read(filename)
+  my_hash = JSON.parse(text)
+  # puts my_hash
```
Comment (Br3nda) [inline dataCleaned.rb]: and this one too

**Review by Br3nda — COMMENTED** [review]
suggest renaming the file. "dataCleaned" doesn't really describe what it does.

**Comment by mermop** [comment]
Thanks @JacOng17 ! I agree with @Br3nda's comments

File: clean_notes_to_json.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+# Command to use to run the script: ./dataCleaned.rb > out.json
```
Comment (Br3nda) [inline clean_notes_to_json.rb]: script was renamed, so the comment here needs updating too

**Review by Br3nda — APPROVED** [review]

**Review by mermop — APPROVED** [review]

**Pull request merged**